pkg/diagnostics/tracing.go

```diff
@@ -31,6 +30,7 @@ const (
 	// CorrelationID is the header key name of correlation id for trace
 	CorrelationID      = "X-Correlation-ID"
 	correlationKey key = CorrelationID
+	headerPrefix       = "dapr-"
```

Comment:
nit: daprHeaderPrefix for readability in the code.

Reply:
done.